nservicebus/azure-service-bus/batching.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Azure Service Bus Transport Batching
+summary: Batching with Azure Service Bus and how it works
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports
+reviewed: 2016-04-26
+---
+
+## Batching store access
+
+In order to increase messages throughput, the Azure Service Bus uses batching prior to writing to its internal store.  That involves delaying write operations. When batching is enabled, the write operations are delayed by 20 ms. Read operations are not affected. This helps to increase the throughput.  
+
+Batching store operations should be disabled for low throughput scenarios requiring low latency. 
+
+The batching is enabled by default. In order to disable it, set `EnableBatching` to `false` on [queues](/nservicebus/azure-service-bus/configuration/configuration.md#Queues), [topics](/nservicebus/azure-service-bus/configuration/configuration.md#Topics), and [subscriptions](/nservicebus/azure-service-bus/configuration/configuration.md#Subscriptions).
+
+
+## Client-side batching
+
+Client-side batching allows to delay sending messages to a queue or a topic for a short period of time. Additional messages sent to the same queue or topic within that period of time, they will be grouped and transmitted together in a single batch. This setting affects the Azure Service Bus Message Senders and can be configured using [`BatchFlushInterval`](/nservicebus/azure-service-bus/configuration/configuration.md#messaging-factories). Client-side batching is enabled by default.
+
```

Comment:
probably something for a guidance document, but when would I turn client side batching off